src/core/js/app.js

```diff
@@ -112,6 +112,13 @@ require([
             outputError(e);
         }
 
+        if (!Adapt.isWaitingForPlugins()) triggerInitialisation();
+        else Adapt.once('plugins:ready', function() {
+            triggerInitialisation();
```

Comment:
@tomgreenfield something odd going on with this PR - most of the commits have been merged. The line you've quoted is out of date (https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/app.js#L87)